File: IT/Interfacce grafiche/Esercizi/Java/Esercizio7.java
/* file Esercizio7.java */

import javax.swing.JFrame;
import javax.swing.BoxLayout;
import javax.swing.JLabel;

public class Esercizio7 {
	public static void main(String [] args) {
		JFrame frame = new JFrame();
		JLabel label1 = new JLabel("Un'etichetta");
		JLabel label2 = new JLabel("Un'altra etichetta");

		frame.getContentPane().setLayout(new BoxLayout(frame.getContentPane(), BoxLayout.Y_AXIS));

		frame.add(label1);
		frame.add(label2);
		frame.pack();

		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
		frame.setVisible(true);
	}
}

File: IT/Interfacce grafiche/Esercizi/Python/esercizio7.py
# file esercizio7.py

try:
	from tkinter import Frame, Label
except ImportError:
	from Tkinter import Frame, Label

frame = Frame()

label1 = Label(frame, text="Un'etichetta")
label2 = Label(frame, text="Un'altra etichetta")

label1.pack()
label2.pack()

frame.pack()
frame.mainloop()

